hummingbird/ml/operator_converters/sklearn/decomposition.py

```diff
@@ -47,6 +49,8 @@ def convert_sklearn_kernel_pca(operator, device, extra_config):
     Returns:
         A PyTorch model
     """
+    assert operator is not None
```

Comment:
I notice we have this `assert` at the start of each converter.  In what case does this fail?  If the model is corrupted?  (Is there some error message that we could give users that would be helpful?)

Reply:
I think this is hard to trigger. I believe it will fail before actually reaching this point. I added them as a double check. I will add an error message. Something like "cannot convert None operator".